src/include/simeng/OS/Process.hh

```diff
@@ -14,6 +14,8 @@
 
 namespace simeng {
 
+static constexpr uint64_t RLIM_INF = 0xffffffffffffffff;
```

Comment:
if they are static consexpr i think they should be in `OS/Constants.hh`

Reply:
moved